src/lib/helpers.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable jsdoc/require-jsdoc */
```

Comment:
In order to reduce the diff noise, this file has been created to host all the helpers that made the migration easier.